source/configure.rst

```diff
@@ -60,11 +60,11 @@ Starting the Quipucords Server
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 After you make the decisions on the configuration options for the server, you can start the Quipucords server. The following commands assume that you used the default port and the recommended steps to create a home directory and subdirectories for the SSH keys, the default options for the PostgreSQL database, and the default log output during the Quipucords server configuration.
 
-If your system does not have SELinux enabled, you can start the Quipucords server with the following Docker command::
+If your system does not have SELinux enabled and you are installing release 1.0.0, you can start the Quipucords server with the following Docker command::
```

Comment:
If your system does not have SELinux enabled, you can start the Quipucords server with the following Docker command, where 1.0.0 is the installed version of Quipucords:

Reply:
👍